native/version

```diff
@@ -1 +1,2 @@
 v0.1.0
+v0.1.0
```

Comment:
Thanks @matiwinnetou . Can you please rollback download_libs.sh and version file from this PR? Otherwise looks good.